src/EnergyPlus/Plant/PlantManager.cc

```diff
@@ -1206,6 +1206,11 @@ void GetPlantInput(EnergyPlusData &state)
                             OutsideEnergySources::OutsideEnergySourceSpecs::factory(state, PlantEquipmentType::PurchHotWater, CompNames(CompNum));
                         break;
                     }
+                    case PlantEquipmentType::PurchSteam: {
+                        this_comp.compPtr =
+                            OutsideEnergySources::OutsideEnergySourceSpecs::factory(state, PlantEquipmentType::PurchSteam, CompNames(CompNum));
+                        break;
```

Comment:
Does this object need to have the `FlowPriority` and `HowLoadServed` variables set to get flow working properly?  I guess maybe not, since the one above it doesn't have it.